src/Utils.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <string>
+using namespace std;
+
+class Utils
```

Comment:
again, no need to use classes as substitutes for namespaces

if all your member functions are static the class should be a namespace